core/src/test/java/org/elasticsearch/fieldstats/FieldStatsIntegrationIT.java

```diff
@@ -169,18 +169,25 @@ public void testRandom() throws Exception {
 
         assertThat(response.getAllFieldStats().get("byte").getMinValue(), equalTo(minByte));
```

Comment:
I'd probably write an `assertFieldStats` method for these three asserts just to cut down on the copy and paste-ness.